### PR TITLE
Redesigned ICARUS TPC PS and PMT HV pages. Added SBND Cathode Drift HV page

### PR DESCRIPTION
**Pull request by giosal**

### CODE REVIEW

File: example_settings_icarus_gateway_sbnd.conf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+CHANNEL_MAP_FNAME="/web/sites/s/sbn-online.fnal.gov/cgi-bin/minargon/minargon/channel_map.txt"
```
Comment (gputnam) [inline example_settings_icarus_gateway_sbnd.conf:1]: Can you remove this from the pull request? It has some internal fermilab information that we should post publicly.

File: minargon/metrics/postgres_api.py.save [hunk 2]
```diff
@@ -0,0 +1,944 @@
+#!/usr/bin/env python
```
Comment (gputnam) [inline minargon/metrics/postgres_api.py.save:1]: Is this file necessary to include in the merge?

File: minargon/metrics/postgres_api.py.save.1 [hunk 3]
```diff
@@ -0,0 +1,944 @@
+#!/usr/bin/env python
```
Comment (gputnam) [inline minargon/metrics/postgres_api.py.save.1:1]: Same here.

File: minargon/static/conf/Sy1527Ech.sub.fnal [hunk 4]
```diff
@@ -0,0 +1,196 @@
+file db/Sy1527ch.tpl{
```
Comment (gputnam) [inline minargon/static/conf/Sy1527Ech.sub.fnal:1]: Is this information that can't be accessed from a database? Does it need to be in a file like this.

File: minargon/static/css/minargon.css [hunk 5]
```diff
@@ -84,19 +84,25 @@ div.alert-holder {
```
Comment (gputnam) [inline minargon/static/css/minargon.css:84]: What do these changes in css do? What are they for?

File: minargon/templates/icarus/layout.html [hunk 6]
```diff
@@ -94,13 +94,13 @@
                   aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
           </button>-->
-          <div class="navbar-collapse collapse" id="navbar">
+          <div class="navbar-collapse collapse bg-maroon" id="navbar">
```
Comment (gputnam) [inline minargon/templates/icarus/layout.html:97]: why add the class?

**Review by gputnam — CHANGES_REQUESTED** [review]

File: minargon/static/conf/Sy1527Ech.sub.fnal (hunk 4 above)
Comment (giosal) [reply minargon/static/conf/Sy1527Ech.sub.fnal:1]: Yes, this is to future-proof the page. This is a mapping of power supply channel to PMT.
The mapping may change in the future, so in order to avoid changing the mapping on the website, the new version of this file can be re-uploaded to replace this file and the mapping on the page will be updated automatically

File: minargon/static/css/minargon.css (hunk 5 above)
Comment (giosal) [reply minargon/static/css/minargon.css:84]: These changes fix the white background in the dropdown menus. At least, this was the solution

File: minargon/templates/icarus/layout.html (hunk 6 above)
Comment (giosal) [reply minargon/templates/icarus/layout.html:97]: I will remove this, not necessary here, it slipped after debugging

File: minargon/metrics/postgres_api.py.save (hunk 2 above)
Comment (giosal) [reply minargon/metrics/postgres_api.py.save:1]: Will remove this file, as it's not necessary. I think this may have been created by nano upon connection time out

File: minargon/metrics/postgres_api.py.save.1 (hunk 3 above)
Comment (giosal) [reply minargon/metrics/postgres_api.py.save.1:1]: same here
